README.md

```diff
@@ -32,13 +32,23 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+    The DOM or Document Object Model is an API that represents an HTML document as a tree structure comprised of node objects.
```

Comment:
To expand on the above a little bit. the DOM serves as an intermediary between the window object and the javascript one would like to use to manipulate elements on the window object.